http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/Boot.scala

```diff
@@ -656,6 +664,50 @@ object Boot extends IOApp {
         googleDependencies.credentials
       )
 
+      val cromwellAppInstall = new CromwellAppInstall[F](
+        ConfigReader.appConfig.azure.coaAppConfig,
+        ConfigReader.appConfig.drs,
+        samDao,
+        cromwellDao,
+        cbasDao,
+        cbasUiDao,
+        azureBatchService,
+        azureApplicationInsightsService
+      )
+      val cromwellRunnerAppInstall =
+        new CromwellRunnerAppInstall[F](ConfigReader.appConfig.azure.cromwellRunnerAppConfig,
+                                        ConfigReader.appConfig.drs,
+                                        samDao,
+                                        cromwellDao,
+                                        azureBatchService,
+                                        azureApplicationInsightsService
+        )
+      val hailBatchAppInstall =
+        new HailBatchAppInstall[F](ConfigReader.appConfig.azure.hailBatchAppConfig, hailBatchDao)
+      val wdsAppInstall = new WdsAppInstall[F](ConfigReader.appConfig.azure.wdsAppConfig,
+                                               ConfigReader.appConfig.azure.tdr,
+                                               samDao,
+                                               wdsDao,
+                                               azureApplicationInsightsService
+      )
+      val workflowsAppInstall =
+        new WorkflowsAppInstall[F](
+          ConfigReader.appConfig.azure.workflowsAppConfig,
+          ConfigReader.appConfig.drs,
+          samDao,
+          cromwellDao,
+          cbasDao,
+          azureBatchService,
+          azureApplicationInsightsService
+        )
+
+      implicit val appTypeToAppInstall = AppInstall.appTypeToAppInstall(wdsAppInstall,
```

Comment:
is this implicit actually used?

Reply:
Yeah, see above -- it's passed to `AKSInterpreter` and used for getting the `AppInstall` instance from an `AppType`.